### ISSUE TITLE
Listen to server emitted events from socket.io server

### ISSUE THREAD

**Issue by sumitnegi7** [ptejada/artillery-engine-socketio-v3#16]
Hi! I am looking  to implement Listen/Receive Events feature on Artillery emitted by my socket.io application. This feature is something I need in my load testing 

@ptejada @jlnquere  will really appreciate a quick response here.

**Comment by ptejada** [comment]
Yes, see https://github.com/ptejada/artillery-engine-socketio-v3/issues/14#issuecomment-1617282189
